_posts/blog/2018-09-10/2018-09-10-the-beginners-guide-to-decentralization.markdown

```diff
@@ -0,0 +1,97 @@
+
+Decentralization has its benefits. There are no gatekeepers preventing you from doing what you want to do. Decentralized systems offer more freedom than centralized systems. There is no single point of failure which makes decentralized systems more resilient. Since there’s no one in charge there is no one that everyone depends on. 
+
+Decentralization also has drawbacks. Designing systems that have no one in control is difficult to do, and if not done properly will mean the system won’t work. Being permissionless does prevent gatekeepers from excluding people, but it’s also difficult (or even impossible) to remove bad actors from the system. Decentralized networks also tend to be more difficult to use than their centralized counterparts.
+
+### The Spectrum of Decentralization
+
+Centralization and decentralization aren’t black and white. It’s not one or the other. Most systems have a mix of centralized and decentralized attributes that fall on a spectrum. When I say “centralized systems” I mean systems that have a clear central point of control. When I say “decentralized systems” I mean systems with no clear central point of control. In practice, there are gray areas. 
+
+For example, a monarchy is a highly centralized form of governance.  However, there are historical examples of monarchs’ power being constrained by other groups, such as aristocracy or religious groups, so even the most centralized form of governance imaginable isn’t fully centralized in practice.
+
+The same is true with decentralized systems. One criticism of “flat hierarchies” within companies (an attempt at eliminating management) is that they remain flat in name only, while some people effectively become managers again over time. Or when computers in decentralized networks communicate peer-to-peer for most communications but are forced to connect to a specific server the first time they start up (called a “bootstrap node”). Maintaining perfect decentralization within a given system is nearly impossible.
+
+It’s also true that the same system can have strong decentralized elements and strong centralized elements at the same time. This can happen in open source software projects. The software itself might be built to communicate in a fully decentralized manner, but the development process itself is completely centralized within a small group or even an individual. There are some companies who offer a platform for users which they fully control and is completely centralized, but their corporate structure itself is decentralized. In these cases, you can’t label the entire system centralized or decentralized, but only certain attributes such as governance or technical architecture.
+
+### The Importance of Decentralization on OpenBazaar
+
+OpenBazaar is unique because of its decentralization, but that alone though isn’t what makes it excellent. What’s important about OpenBazaar being decentralized is _what it enables users to do_.
```

Comment:
The idea that OpenBazaar is excellent kind of comes out of left field here. While it may in fact be excellent, the concept isn't connected to anything in the post before this.

Should there be something in the first paragraph about OB being great, not just decentralized?